cypress/spec/test.spec.ts

```diff
@@ -0,0 +1,8 @@
+describe('test', () => {
+  it('first test', () => {
+    cy.visit('http://localhost:3000');
```

Comment:
It will be better to use the environment variable for this link from the .env file. You can check [this library](https://www.npmjs.com/package/dotenv)